src/example-types/metricGroups/dateRangesGroupStats.js

```diff
@@ -1,17 +1,41 @@
 let _ = require('lodash/fp')
 let { groupStats } = require('./groupStatUtils')
 
-let buildGroupQuery = ({ field, ranges }, children, groupingType) => ({
-  aggs: {
-    [groupingType]: {
-      date_range: { field, ranges },
-      ...children,
+let splitDrilldown = (drilldown, fn = _.identity) =>
+  _.map(fn, _.split(/(?<=Z)-/, drilldown))
+
+let buildGroupQuery = (
+  { field, ranges },
+  children,
+  groupingType,
+  _schema,
+  _getStats,
+  drilldownKey
```

Comment:
Since the prop list is growing and some are less relevant, I wonder if it's time to put some of these in an object instead of a list. Might be out scope for this PR, but as a general rule we try to avoid super long prop lists for functions and switch passing an object when it gets unwieldy - especially in situations like these where you're ignoring a bunch of the props 🤔

Reply:
Yeah I agree I had that thought, but I also thought refactoring everywhere this function is called was out of scope for the PR.  How about we do it in a follow-up?  I definitely agree it's time to change it to an object parameter though.

Reply:
Sounds good